### DISCUSSION TITLE
Other frameworks installation instructions

### DISCUSSION THREAD

**Discussion by benjaminpreiss** [payloadcms/payload#10040] [Q&A]
Hey everyone,

currently digging through the docs, especially the installation instructions, but it is not clear to me right now how I can install payload when not using Next.js.

I was thinking of setting up a pnpm workspace with payload living next to my SvelteKit app in the `apps/` folder.

Looking forward to your ideas!

**Comment by rilrom** [comment]
It's not an official example but this repo could be useful:

https://github.com/fusionary/turbo-payload/tree/main
